sphinx_immaterial/apidoc/cpp/apigen.py

```diff
@@ -43,19 +43,22 @@ class ApigenConfig:
     document_prefix: str
     """Sphinx document path prefix for the per-entity documentation pages.
 
-    This should end in :python:`"/"`.
+    .. important::
+        Because this is used as a Regular Expression pattern, this should end in
+        (and exclusively use) the Linux path separator :python:`"/"` (even if
+        building on Windows).
```

Comment:
I'm not sure how relevant this is here. I don't think this is used as a regexp, but it does seem that Unix path separators are preferred.

Reply:
Yes, that is true --- I think this was an accidental change.

Reply:
Yep. It was meant to go in `dis/allow_paths`, but I forgot to remove it after realizing I added it to the wrong file.

Should I alter the admonition? Or should I remove the admonition and make sure the path separators use the Unix form?

Reply:
Probably alter it.

Reply:
altered it.